evaluation.md

```diff
@@ -2,18 +2,47 @@
 
 ## Internal DSL
 
-_Describe each change from your ideal syntax to the syntax you implemented, and
-describe_ why _you made the change._
+**Describe each change from your ideal syntax to the syntax you implemented, and
+describe_ why _you made the change.**
+
+First, we changed "def" to "state", because def is a reserved keyword in scala, so we couldn't use it.
+
+Next, we got rid of colons after the state name. We tried to make these into a function, but we decided to just use the apply function.
```

Comment:
I think single colons are just not valid identifiers in Scala.  Funnily enough, in one instance in my internal DSL I also replaced something I had wanted to be a colon with an apply function.